packages/snap/src/index.ts

```diff
@@ -155,7 +155,9 @@ export const onRpcRequest = async ({
         throw new Error('User rejected request');
       }
 
-      return Promise.all(vcs.map((vc) => vcManager.clearMatchedVcs(vc)));
+      await Promise.all(vcs.map((vc) => vcManager.clearMatchedVcs(vc)));
+
+      return undefined;
```

Comment:
This is more idiomatic

```suggestion
      return;
```